code/game/machinery/vending.dm

```diff
@@ -1150,6 +1165,8 @@
 	contraband = list(/obj/item/reagent_containers/pill/tox = 3,/obj/item/reagent_containers/pill/stox = 4,/obj/item/reagent_containers/pill/antitox = 6)
 	idle_power_usage = 211 //refrigerator - believe it or not, this is actually the average power consumption of a refrigerated vending machine according to NRCan.
 	auto_price = FALSE
+	custom_vendor = TRUE // Chemists can load it for MDs
```

Comment:
You sure money from sales won't go to the Guild? Same question for `/wallmed`.

Reply:
Good catch 👍